mycelium/src/router.rs

```diff
@@ -1193,7 +1193,7 @@ where
                     || new_route.metric().delta(&old_route.metric()) > BIG_METRIC_CHANGE_TRESHOLD
             }
             (None, Some(new_route)) => {
-                info!(
+                debug!(
```

Comment:
I understand that the logs right now are a bit spammy, though given the current stage of development I am fine with that. This will happen at some point, but there need to be some other changes first, and for those having this as info logs will be beneficial.

For now, you should be able to disable router logs by adding a specific level filter for `mycelium::router` set to Warn or Error when you create the logger

Reply:
> For now, you should be able to disable router logs by adding a specific level filter for mycelium::router set to Warn or Error when you create the logger

OK, will do

Reply:
> and for those having this as info logs will be beneficial.

On the second thought, i think i will leave it as is for now.
Maybe i can have some info from the logs as well.